frontends/parsers/p4/p4parser.ypp

```diff
@@ -1260,6 +1261,7 @@ optInitializer
 
 initializer
     : expression                          { $$ = $1; }
+    | "..."                               { $$ = new IR::ListExpression(@1, *(new IR::Vector<IR::Expression>()), true); }
```

Comment:
I am a little uncomfortable that this is a list expression. What if this is assigned to a scalar?
(Do we allow scalars to be initialized this way? `bit<32> b = ...;`

Reply:
This solution doesn't allow default initializer for scalar types. Using `...` makes initializing structs, headers and tuples easier and notation shorter, but I don't see any benefits of using it for scalar types. If you think it has a use case I can try to implement it for scalar types too.